File: youtubejextractor/src/main/java/com/github/kotvertolet/youtubejextractor/models/test/Accessibility.java
package com.github.kotvertolet.youtubejextractor.models.test;

import com.google.gson.annotations.SerializedName;

public class Accessibility{

	@SerializedName("label")
	private String label;

	public void setLabel(String label){
		this.label = label;
	}

	public String getLabel(){
		return label;
	}
}

File: youtubejextractor/src/main/java/com/github/kotvertolet/youtubejextractor/models/newModels/Accessibility.kt
package com.github.kotvertolet.youtubejextractor.models.newModels

import android.os.Parcelable
import com.google.gson.annotations.SerializedName
import kotlinx.android.parcel.Parcelize
import java.io.Serializable

@Parcelize
class Accessibility : Parcelable, Serializable {
    @SerializedName("label")
    var label: String? = null
    override fun toString(): String {
        return "Accessibility{" +
                "label = '" + label + '\'' +
                "}"
    }
}